src/test/shell/bazel/tags_propagation_native_test.sh

```diff
@@ -272,125 +272,4 @@ EOF
   assert_contains_n "no-remote:" 1 output1
 }
 
-# Test a native test rule which has tags, that should be propagated (independent of flags)
```

Comment:
this comment was outdated, likely just since starlarkification, this test was relying on the off behavior as well